src/app/notification/notification.component.html

```diff
@@ -1,9 +1,29 @@
 <div
   class="notification"
+  *ngIf="!notification.choices"
   [ngClass]="['notification__border-' + notification.type, show ? 'notification__show' : '']"
   (click)="hideNotification()"
 >
   <span class="notification__heading">{{ notification.heading }}</span>
   <span class="notification__text">{{ notification.text }}</span>
   <span class="notification__close" i18n="@@tap-close">tap this card to close it</span>
 </div>
+
+<div
+  class="notification-prompt"
+  *ngIf="notification.choices?.length > 0"
```

Comment:
Wouldn't it be enough to check whether `notification.choices` exist here?

Reply:
this is just in case an empty list is sent, we would not want to display a prompt without choices